multimedia/templates/multimedia/movie_detail.html

```diff
@@ -0,0 +1,6 @@
+{% extends "multimedia/base_detail.html" %}
+
+{% block description %}
+<h3>{{ movie.name }}</h3>
+<pre>{{ movie.description }}</pre>
```

Comment:
Preformatted Text. If It is `<p>` then new line won't be printed, unless every new line is wrapped with `<p>`